
// 定义消息ID
enum MessageID {
    MSG_TIME_TICK   = 0;    // 时钟tick
    MSG_REQ_LOGIN   = 1;    // 请求登陆
}

// 消息包装层
message RawMessage {
    required MessageID  id      = 1 [default = MSG_TIME_TICK];  // 消息ID。默认为tick
    required string     strMsg  = 2;  // 消息内容。需要进一步解包
}
